Components/Bites/src/OgreApplicationContextSDL.cpp

```diff
@@ -68,7 +68,22 @@ NativeWindowPair ApplicationContextSDL::createWindow(const Ogre::String& name, O
     p.miscParams["sdlwin"] = Ogre::StringConverter::toString(size_t(ret.native));
 
 #if OGRE_PLATFORM == OGRE_PLATFORM_LINUX
-    p.miscParams["externalWindowHandle"] = Ogre::StringConverter::toString(size_t(wmInfo.info.x11.window));
+    if (wmInfo.subsystem == SDL_SYSWM_WAYLAND)
+    {
+#ifdef OGRE_WAYLAND
```

Comment:
I just realized that everything can be done at runtime here and we can move the assert to GLSupport

Reply:
I understand. My latest commit does this.